packages/core/src/Button/Button.js

```diff
@@ -94,6 +95,7 @@ export const buttonStyles = css`
   text-align: center;
   text-decoration: none;
   font-family: inherit;
+  font-weight: 600;
```

Comment:
Is this where we wanted to get `fontWeight` from `theme`?

Reply:
bold is 700 though, what do you think?

Reply:
We don't even bring in the 600 font weight, at least on the landing page so I don't think changing it to 700 is an issue because it is already using that when set to 600
![image](https://user-images.githubusercontent.com/62619213/99106134-59ba1480-25a9-11eb-8c1f-56cc493b22e5.png)

Reply:
yea i would change it to the theme.fontWeights that maps to 700